tools/info/main.cpp

```diff
@@ -632,14 +634,17 @@ int main(int argc, const char** argv)
             // GFXRECON_WRITE_CONSOLE("\nDraw/dispatch call info:");
             // GFXRECON_WRITE_CONSOLE("\tTotal draw calls: %" PRIu64, stats_consumer.GetDrawCount());
             // GFXRECON_WRITE_CONSOLE("\tTotal dispatch calls: %" PRIu64, stats_consumer.GetDispatchCount());
+
+            GFXRECON_WRITE_CONSOLE("\n");
```

Comment:
Is this extra newline intentional?

Reply:
https://github.com/LunarG/gfxreconstruct/blob/dev/tools/info/main.cpp#L494

Reply:
I just replaced this to
```
if (file_processor.GetCurrentFrameNumber() == 0)
{
    GFXRECON_WRITE_CONSOLE("\nFile did not contain any frames");
}
```

Reply:
https://github.com/LunarG/gfxreconstruct/pull/864/files#diff-8b3e734e213d80820573b5bdb3c821349c5387e09b96ad221f1dee67f000070fR638